src/ng/raf.js

```diff
@@ -24,8 +24,42 @@ function $$RAFProvider() { //rAF
           };
         };
 
-    raf.supported = rafSupported;
+    queueFn.supported = rafSupported;
 
-    return raf;
+    var cancelLastRAF;
+    var taskCount = 0;
+    var taskQueue = [];
+    return queueFn;
+
+    function flush() {
+      for (var i = 0; i < taskQueue.length; i++) {
+        var task = taskQueue[i];
+        if (task) {
+          taskQueue[i] = null;
+          taskCount--;
```

Comment:
But here you are running a loop that decreases `taskCount` and right after that you srt it to `0`.
So, decrementing it is redundant.